src/tree/updater_gpu_hist.cu

```diff
@@ -69,10 +87,10 @@ __device__ GradientPairSumT ReduceFeature(common::Span<const GradientPairSumT> f
 
 /*! \brief Find the thread with best gain. */
 template <int BLOCK_THREADS, typename ReduceT, typename scan_t,
-          typename max_ReduceT, typename TempStorageT>
+          typename max_ReduceT, typename TempStorageT, typename GradientSumT>
```

Comment:
The linter didn't catch it, but since we are here and there are some formatting changes in this PR, we can also rename the `max_ReduceT` to something like `MaxReduceT`.